tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -66,8 +66,7 @@ Define {
 	HDMI1_ID			3
 	HDMI2_ID			4
 	HDMI3_ID			5
-	HDMI4_ID			6
-	SPK_ID				7
+	SPK_ID				6
```

Comment:
Won't changing SPK_ID impact the existing topologies, like sof-mtl-max98357a-rt5682.tplg?

Reply:
new commit. SPK_ID changes only for sof-mtl-rt1019-rt5682.tplg